src/librustdoc/html/layout.rs

```diff
@@ -99,7 +99,7 @@ crate fn render<T: Print, S: Print>(
                            placeholder=\"Click or press ‘S’ to search, ‘?’ for more options…\" \
                            type=\"search\">\
                 </div>\
-                <button type=\"button\" class=\"help-button\">?</button>
+                <button type=\"button\" id=\"help-button\">?</button>
```

Comment:
It broke the style of the button (check on the dark theme). You need to update the CSS as well. ;)